Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/Response/Json/JsonResponseData.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Hunted_Mobile.Model.Response.Json {
+    public interface JsonResponseData {
```

Comment:
Waarvoor een `interface` als er vervolgens niks mee wordt gedaan? Wat is het nut dan van een interface?

Reply:
Dit interface en `ResponseData` gebruik ik om bepaalde klassen en structs mee te markeren om generics in de JSON services robuuster te maken. Er is alleen geen concrete data of gedrag dat al deze klassen gemeen hebben, dus zit er verder niks in het interface.